catalog-info.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: habitat
+  description: Casavo Design System
+tags:
+  language: Typescript
```

```suggestion
  - language:typescript
  - npm 
```

this is an array of tags (i.e. strings, with some [specific limits](https://backstage.io/docs/features/software-catalog/descriptor-format#tags-optional))